guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: guide
+title: Transaction privacy
+description: An overview of how to help users maintain their financial privacy while using Bitcoin.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/privacy/privacy-preview.jpg
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/privacy/header.jpg"
+    retina = "/assets/images/guide/payments/privacy/header@2x.jpg"
+    mobile = "/assets/images/guide/payments/privacy/privacy-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/privacy/privacy-header-mobile@2x.jpg"
+    alt-text = "Illustration of an eye, with a cross on top of an ink blot background"
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+
+# Transaction privacy
+
+<!--
+This page should inform about what information is made public when sending or receiving, how the application can help minimize unnecessary privacy leaks, basic application functionality to help, and when we can, more advanced options.
+- Single-use addresses (avoiding address reuse)
+- Coin selection / labelling
+- Coin join / Pay join
+-->
+
+It’s a common misconception that Bitcoin is anonymous. Instead, it should be referred to as pseudonymous which means each address you generate is like a new identity. Unless you reveal that you are the owner whether directly or indirectly you are able to remain anonymous.
```

Comment:
```suggestion
A common misconception is that Bitcoin payments are anonymous. Rather, Bitcoin payments are pseudonymous. This means each new bitcoin address acts like a new identity. 

These addresses can have a users real identity tied to them. This can happen willingly, say by sharing an address on your Facebook profile, or unwillingly, through block chain analysis tools.
```

Some copy suggestions. Should keep the focus on this being privacy in the context of payments - not just general Bitcoin privacy.

Reply:
I was trying to correct "users" to "user's" but it seems to have messed up the commit suggestion, sorry :(

Reply:
All good, think I fixed it